src/server/routes/units.js

```diff
@@ -104,9 +105,9 @@ router.post('/edit', async (req, res) => {
 			await unit.update(conn);
 		} catch (err) {
 			log.error('Failed to edit unit', err);
-			res.status(500).json({ message: 'Unable to edit units.', err });
+			failure(res, 500, 'Unable to edit units ' + req.body.identifier);
```

Comment:
I think this line was fine with returning the err. It was the next one so the success uses req.body.identifier instead of id.